test/shell/014_add_nonexistent.sh

```diff
@@ -7,4 +7,4 @@ touch cfbs.json && rm cfbs.json
 
 cfbs init
 ! ( cfbs add bollocks > output.log 2>&1 )
-grep -F "Error: Module 'bollocks' does not exist" output.log
+grep -F "Error: Module 'bollocks' version 'latest' does not exist" output.log
```

Comment:
IMO, this error message was nicer before and we could make it only print version in case it's not latest.

Reply:
Agreed 👍